npm-debug.log

```diff
@@ -0,0 +1,48 @@
+0 info it worked if it ends with ok
```

Comment:
I think this log can be added to gitignore

Reply:
i will added it now